### ISSUE TITLE
Upgrading to latest synapse+pantalaimon+slack fails 'must support MSC2666'

### ISSUE THREAD

**Issue by JohnStarich** [matrix-org/matrix-appservice-slack#689]
**Describe the bug**
To be honest, I'm quite lost on which component isn't working, and I've searched through the MSCs, open bugs, and all 3 relevant repos. I'll do my best to pull relevant logs, please let me know if we need more.

My slack bridge container starts up, dumps these logs, and then shuts down again with encryption enabled:
```
(node:10) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 31 finish listeners added to [Console]. Use emitter.setMaxListeners() to increase limit
Jul-24 07:50:25.118 INFO SlackHookHandler Slack-side listening on port 9898 over http
Jul-24 07:50:25.137 INFO bridge [-] GET http://pantalaimon:8008/_matrix/client/r0/login (AS) Body:
Jul-24 07:50:28.350 INFO bridge [-] GET http://pantalaimon:8008/_matrix/client/r0/login (AS) HTTP 200 "{\"flows\":[{\"type\":\"m.login.password\"},{\"type\":\"com.devture.shared_secr
Jul-24 07:50:28.363 ERROR app Failed to start: Error: To enable support for encryption, your homeserver must support MSC2666
    at Bridge.checkHomeserverSupport (/usr/src/app/node_modules/matrix-appservice-bridge/lib/bridge.js:997:23)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Bridge.initalise (/usr/src/app/node_modules/matrix-appservice-bridge/lib/bridge.js:224:9)
    at async Bridge.run (/usr/src/app/node_modules/matrix-appservice-bridge/lib/bridge.js:302:9)
    at async Main.run (/usr/src/app/lib/Main.js:835:9)
e2c: exit status 1
```

**To Reproduce**
The steps are a bit fuzzy, unfortunately. Let me know what I can fetch to help debug a bit more.

Container image versions (all latest available on Docker Hub):
* synapse: 1.63.1
* pantalaimon: 0.10.4
* appservice-slack: 1.11.0

To reproduce:
1. register the bridge (I configured the previously used registration file)
2. configure appservice-slack with encryption enabled and pointed at pantalaimon
3. boot up all 3
4. observe appservice-slack shuts down immediately

Unfortunately, I did not have a clean environment to test. If needed, I can try to spin one up. I'm crossing my fingers there's an obvious user error! Happy to dig in as needed.

Disabling encryption did unstick the boot up process, but now it's struggling with "not invited to room". 😓 

**Expected behavior**
I expected the bridge to boot successfully with encryption enabled.

**Screenshots**
See above log snippet ^

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
I did upgrade from a fairly old synapse version (1.36.0) to the latest (1.63.1). There's a possibility I foobared the upgrade with respect to bridges.

Previous versions:
* synapse: 1.36.0
* pantalaimon: 0.10.0
* appservice-slack: 1.8.0

**Comment by Half-Shot** [comment]
Hiya, this is on our radar! It's a known bug that is going to be fixed in #690

**Comment by JohnStarich** [comment]
Thanks @Half-Shot! Is there a specific Docker tag (or maybe other release?) I should use to verify?
